views/components/user-navigation/macro.njk

```diff
@@ -13,12 +13,23 @@
                     {% if previousPage %}
                         <li class="user-nav__pagination-link user-nav__pagination-link--prev">
                             <span class="user-nav__pagination-arrow">←</span>
-                            <input class="btn-link" name="previousBtn" type="submit" formaction="" value="{{ __('applyNext.navigation.previous') }}"/>
+                            <div class="user-nav__pagination-btn-container">
```

Comment:
Do we need this extra `user-nav__pagination-btn-container` div?

Reply:
Yup (or a `span`). It was a solution for steps label not wrapping below the next/prev buttons (earlier it was within `<a>` element).